collect_stata/write_json.py

```diff
@@ -0,0 +1,369 @@
+"""write_json.py"""
+__author__ = "Marius Pahl"
+
+import json
+from collections import Counter, OrderedDict
+
+import numpy as np
+import pandas as pd
+
+
+def sorting_dataframe(values, labels, missings, frequencies):
+    """Method to sort values and labels and return sorted dict"""
+    dataframe = pd.DataFrame(
+        {
+            "values": values,
+            "labels": labels,
+            "missings": missings,
+            "frequencies": frequencies,
+        }
+    )
+    dataframe["labels"] = dataframe["labels"].astype(str)
+    dataframe["values"] = pd.to_numeric(dataframe["values"])
+    dataframe.sort_values(by="values", inplace=True)
+    return dataframe.to_dict("list")
+
+
+def uni_cat(elem, file_csv):
+    """Generate dict with frequencies and labels for categorical variables
+
+    Input:
+    elem: dict
+    file_csv: pandas DataFrame
+
+    Output:
+    cat_dict: dict
+    """
+
+    frequencies = []
+    values = []
+    missings = []
+    labels = []
+
+    stata_missings = {
```

Comment:
Is this necessary with the recent version of pandas?

Reply:
Please find another way to handle this overflow, if it still needs handling.

Reply:
see #23